### ISSUE TITLE
Create Partial Model with fields excluded

### ISSUE THREAD

**Issue by ClayMav** [joshorr/pydantic-partials#24]
For my use case, I am using a partial model as the input type to a patch operation. I am heavily utilizing openapi schemas and want the input type to my function to contain all minus a few fields from the data model. Below is an example to demonstrate.
```
class MyModel(BaseModel):
    name: str
    age: int
    create_time: datetime
    update_time: datetime
    id: str

class PartialMyModel(MyModel, PartialModel): # I want to exclude create_time, update_time, and id
    pass
```
Ideally, I should be able to define a class or list of fields to exclude and pass that in so that I can remove those fields on all my models that I use in this way.

**Comment by joshorr** [comment]
@ClayMav Do you mean exclude them from being partials?  Or are you meaning something else?

Right now you can disable the automatic partials option and then individual mark specific fields, but that's an inclusive style.  I did not think about an exclusive style, where you wanted to exclude specific fields from being made partials.

I think that would be fairly easy to implement.  Perhaps I can try and do something something this week.

Sorry it took a while to get back to you!  I just did not notice the issue until just now (I'll have to look at my email notifications).

**Comment by joshorr** [comment]
I'll try to get this done this weekend, we will see what happens 😋

**Comment by joshorr** [comment]
@ClayMav I have done the work and it's released as v1.1.0.  You can look at the documentation here in the readme (https://github.com/joshorr/pydantic-partials?tab=readme-ov-file#exclude-fields-from-auto-partials).  Let me know if you have any questions.

https://github.com/joshorr/pydantic-partials/pull/25
https://github.com/joshorr/pydantic-partials/pull/26

**Comment by joshorr** [comment]
Finished and released as `v1.1.0`
